### PR TITLE
fix: use autorange when metric chart is set to log scale

### PR DESCRIPTION
**Pull request by hkang1**
## Description

Fix issue of chart not setting the y-axis scale properly when viewing charts on a log scale.

Linear Scale
Screen Shot 2020-09-14 at 5 46 43 PM

Log Scale
Screen Shot 2020-09-14 at 5 46 37 PM

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/react/src/components/MetricChart.tsx [hunk 1]
```diff
@@ -134,6 +134,7 @@ const MetricChart: React.FC<Props> = (props: Props) => {
     layout.xaxis.title = props.xLabel;
     layout.yaxis.title = props.yLabel;
     layout.yaxis.type = scale;
+    layout.yaxis.autorange = scale === Scale.Log;
     layout.xaxis.range = range ? range.xaxis : (maxRange ? maxRange.xaxis : undefined);
     layout.yaxis.range = range ? range.yaxis : (maxRange ? maxRange.yaxis : undefined);
```
Comment (hamidzr) [inline webui/react/src/components/MetricChart.tsx:139]: question: would we want to also avoid manually setting the yaxis range when we are using the log scale or it doesn't matter

**Review by hamidzr — APPROVED** [review]
LGTM

**Comment by hamidzr** [comment]
I run into an issue with resetting the range when zooming back out (double click) but it also happens in master in linear so unrelated?

**Comment by hkang1** [comment]
looking into `Plotly.relayout` to see if it renders the log scale properly

**Comment by hkang1** [comment]
This approach doesn't seem to fix all the rendering issues around log scale.

**Pull request closed without merging**